scripts/run_tutorials.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import json
+import os
+import time
+from pathlib import Path
+
+import papermill
+from nbclient.exceptions import CellTimeoutError
+
+TUTORIALS_TO_SKIP = [
+    "raytune_pytorch_cnn",  # TODO: Times out CI but passes locally. Investigate.
+]
+
+
+def run_script(
+    tutorial: Path, timeout_minutes: int, env: dict[str, str] | None = None
+) -> None:
+    if env is not None:
+        os.environ.update(env)
+    papermill.execute_notebook(
+        tutorial,
+        tutorial,
+        # This timeout is on cell-execution time, not on total runtime.
+        execution_timeout=timeout_minutes * 60,
+    )
```

Comment:
We should probably unify this setup with BoTorch as well (uses subprocess here IIRC), but that's not for this PR.